models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
+  family : string
+  given : string
+  suffix : string
+  phone : {
+    value : string
+    use : enum (home, work, temp, old, mobile)
+  }
+  email : array of strings
+  gender : enum (male, female, other, unknown)
+  birthdate : string
+  race : enum (American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Black or African-American, White, Other Race)
+  ethnicity: enum (Hispanic, Not Hispanic)
+  permissionToFollowup: boolean
```

Comment:
I don't think we need "permissionToFollowup" anymore (we are changing the UI to only collect the phone and email contact if permission is given)

Reply:
Thanks